storage/encryption.js

```diff
@@ -55,35 +55,26 @@ function generateEncryptionKey () {
  * The file will be encrypted by Google Cloud Storage and only retrievable using
  * the provided encryption key.
  *
- * @param {string} bucketName The bucket where the file will be uploaded.
- * @param {string} srcFileName The local file to be uploaded.
- * @param {string} destFileName The name of the destination file.
- * @param {string} key The encryption key.
+ * @param {object} options Configuration options.
+ * @param {string} options.bucket The bucket where the file will be uploaded.
+ * @param {string} options.srcFile The local file to be uploaded.
+ * @param {string} options.destFile The name of the destination file.
+ * @param {string} options.key The encryption key.
  * @param {function} callback The callback function.
  */
-function uploadEncryptedFile (bucketName, srcFileName, destFileName, key, callback) {
-  if (!bucketName) {
-    return callback(new Error('"bucketName" is required!'));
-  } else if (!srcFileName) {
-    return callback(new Error('"srcFileName" is required!'));
-  } else if (!destFileName) {
-    return callback(new Error('"destFileName" is required!'));
-  } else if (!key) {
-    return callback(new Error('"key" is required!'));
-  }
-
-  var bucket = storage.bucket(bucketName);
-  var options = {
-    destination: destFileName,
-    encryptionKey: new Buffer(key, 'base64')
+function uploadEncryptedFile (options, callback) {
+  var bucket = storage.bucket(options.bucket);
+  var config = {
+    destination: options.destFile,
+    encryptionKey: new Buffer(options.key, 'base64')
   };
 
-  bucket.upload(srcFileName, options, function (err, file) {
+  bucket.upload(options.srcFile, config, function (err, file) {
     if (err) {
       return callback(err);
     }
 
-    console.log('Uploaded encrypted file: %s', destFileName);
+    console.log('Uploaded gs://%s/%s', options.bucket, options.destFile);
```

Comment:
I think the method name (`uploadEncryptedFile`) and CLI (`node encryption upload ...`) cover it.